addons/website/static/src/components/dialog/dialog.js

```diff
@@ -38,6 +39,21 @@ export class WebsiteDialog extends Component {
             disabled: false,
         });
         this.modalRef = useChildRef();
+
+        if (this.props.validateOnEnter) {
+            useEffect((el) => {
+                if (!el) {
+                    return;
+                }
+                const onKeydown = async (ev) => {
+                    if (ev.key === "Enter" && !ev.target.closest(".modal-footer")) {
+                        this.props.validateOnEnter();
+                    }
+                };
+                el.addEventListener("keydown", onKeydown);
```

Comment:
I am not convinced by keydown, keyup would seem like a better candidate:
- just functionally I find it weird that something happens while I did not release my button/mouse yet
- if you keep your keypressed, keydown is triggered in a loop
- if some code is happening on the `change` event, it will be ignored, as keydown comes first (although, I am not sure keyup does come after, to check)